build.gradle

```diff
@@ -77,3 +81,8 @@ eclipse {
   }
 }
 
+// see https://github.com/srs/gradle-node-plugin/blob/master/docs/grunt.md
+grunt_default.dependsOn 'npmInstall'
+// to delete all js file which are not needed you should use findMainClass
```

Comment:
Could you please describe how the deletion works? Is it part of the plugin? I don't get it.

Reply:
The main fact is that you first need to start the app with springBoot and then grunt to get the **app.min.js**. If you don't to that all js files will be copied by **processResources** and no app.min.js is available. ProcessResources overrides all files.  The plugin helps to run grunt within gradle and in phases like **findMainClass**. At this time all resources are copied. Grunt can do his job and removes all unneeded js files by the **clean** task . So the deletion is not part of the plugin. It just helps grunt can do it.